src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -250,6 +250,14 @@
     @Deprecated @Restricted(NoExternalUse.class)
     public transient String server;
 
+    /**
+     * whether to verify ldaps sever certificate? default is false
+     */
+    @SuppressFBWarnings(value = "UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD",
+            justification = "This public field is exposed to the plugin's API")
+    @Deprecated @Restricted(NoExternalUse.class)
+    public transient boolean sslVerify;
```

Comment:
You are adding a new deprecated, transient field, meaning it should not be used and will not be stored on disk.

I also don't think this should be set on the global level but on each separate configuration.
The default should also be `true` since that is the old behavior. And you need to make sure that old configurations are set to `true` when upgrading the plugin, so that Jenkins doesn't suddenly start behaving differently (less secure) in that sense.

Reply:
New commit resolved. Please help review.